src/cli/onefuzz/backend.py

```diff
@@ -241,7 +241,7 @@ def request(
     ) -> Any:
         if not self.config.endpoint:
             raise Exception("endpoint not configured")
-        url = self.config.endpoint + "/api/" + path
+        url = "https://" + self.config.endpoint + "/api/" + path
```

Comment:
i think you will end up having http twice in the url with this change

Reply:
yeah, possible. I'm just patching things to get something going. Previous change was still failing